unstructured_inference/inference/ordering.py

```diff
@@ -31,3 +34,51 @@ def order_layout(
     # NOTE(alan): Temporarily revert to orginal logic pending fixing the new logic
     # See code prior to this commit for new logic.
     return layout
+
+
+def order_two_column_page(elements: List[LayoutElement], width_page: int) -> List[LayoutElement]:
+    """Order the elements of a page with two columns."""
+    # Split the image vertically
+    vertical_line_x = width_page / 2
+
+    # Determine the order of the bounding boxes
+    left_boxes = []
+    right_boxes = []
+    both_sided_boxes = []
+
+    new_bounding_boxes_ix = []
+    for i, bbox in enumerate(elements):
+        x_min = bbox.x1
+        x_max = bbox.x2
+        if x_min < vertical_line_x and x_max < vertical_line_x:
+            left_boxes.append(bbox)
+        elif x_min > vertical_line_x and x_max > vertical_line_x:
+            right_boxes.append(bbox)
+        else:
+            both_sided_boxes.append(bbox)
```

Comment:
I'm worried things like page numbers, headers, and footers would get classified as left boxes and right boxes and get put in a weird order.

Reply:
We might need to identify margins to avoid that. Not that that's easy.

Reply:
Yeah, that elements will be categorized as any of those, but ... where should be placed? They aren't in a column

Reply:
Off the top of my head, I'm thinking headers first, then body text by column, with left and right margin stuff included as body text (this isn't quite right for the arxiv watermarks in the left margin, which is more like header text, but I'm thinking of cases where margin text is used for notes on the text it's adjacent to, and I think including it inline would work), then footers last. Page numbers included in the header/footer is ok I think.